pkg/master/master.go

```diff
@@ -669,28 +661,36 @@ func (m *Master) thirdpartyapi(group, kind, version string) *apiserver.APIGroupV
 	}
 }
 
+func (m *Master) GetRESTOptionsOrDie(c *Config, resource unversioned.GroupResource) generic.RESTOptions {
+	storage, err := c.StorageDestinations.Get(resource)
+	if err != nil {
+		glog.Fatal(err.Error())
+	}
+
+	return generic.RESTOptions{
+		Storage:                 storage,
+		Decorator:               m.StorageDecorator(),
+		DeleteCollectionWorkers: m.deleteCollectionWorkers,
+	}
+}
+
 // getExperimentalResources returns the resources for extensions api
 func (m *Master) getExtensionResources(c *Config) map[string]rest.Storage {
-	restOptions := func(resource string) generic.RESTOptions {
```

Comment:
Oh, this is where my comment was. Yes, I saw that, my suggestion was for readability. `restOptions("deployments")` is better than `m.GetRESTOptionsOrDie(c, extensions.Resource("deployments"))`